README.md

```diff
@@ -16,14 +24,11 @@ using the FLEXMEASURES_PLUGIN_PATHS setting.
 2. Add ENTSOE_AUTH_TOKEN to your FlexMeasures config (e.g. ~/.flexmeasures.cfg).
 You can generate this token after you made an account at ENTSO-E, read more [here](https://transparency.entsoe.eu/content/static_content/Static%20content/web%20api/Guide.html#_authentication_and_authorisation). 
 
-3. pip install entsoe-py
+3. Optionally, edit other settings (default shown here):
+  - ENTSOE_COUNTRY_CODE ("NL")
+  - ENTSOE_TIMEZONE ("Europe/Amsterdam")
```

Comment:
I suggest renaming this to ENTSOE_COUNTRY_TIMEZONE. At least I think that is what is meant.